src/components/a-painter.js

```diff
@@ -1,7 +1,12 @@
-/* global AFRAME saveAs Blob uploadcare */
+/* global AFRAME Blob uploadcare */
+
+var saveAs = require('../../vendor/saveas.js').saveAs;
+
 AFRAME.APAINTER = {
-  version: 1,
-  brushes: {},
+  version: 1
+};
+
+AFRAME.registerComponent('a-painter', {
   sceneEl: null,
   init: function () {
     this.sceneEl = document.querySelector('a-scene');
```

Comment:
If you use `registerSystem`, `this.sceneEl` will already be set for you, and we can remove the `sceneEl: null`.